README.md

```diff
@@ -1,3 +1,6 @@
+# Lucy note
+It's the addon but with extra stuff. The author of the add-on refused to accept my pull request so here we are.
```

Comment:
That was not a refusal, consider pinging him again and adding him as assignee/reviewer